src/StakeRegistry.sol

```diff
@@ -224,32 +246,37 @@ contract StakeRegistry is StakeRegistryStorage {
     }
 
     function setMinimumStakeForQuorum(
-        uint8 quorumNumber, 
+        uint8 quorumNumber,
         uint96 minimumStake
     ) public virtual onlyCoordinatorOwner quorumExists(quorumNumber) {
         _setMinimumStakeForQuorum(quorumNumber, minimumStake);
     }
 
     /**
-     * @notice Sets the stake type for the registry
+     * @notice Sets the stake type for the registry for a specific quorum
+     * @param quorumNumber The quorum number to set the stake type for
      * @param _stakeType The type of stake to track (TOTAL_DELEGATED, TOTAL_SLASHABLE, or BOTH)
      */
-    function setStakeType(StakeType _stakeType) external onlyCoordinatorOwner {
-        _setStakeType(_stakeType);
+    function setStakeType(uint8 quorumNumber, StakeType _stakeType) external onlyCoordinatorOwner {
```

Comment:
feels kinda unsafe setting without lookahead, not too opinionated tho. a comment would be good

Reply:
based on our other comments we can align to have overloaded functions that have unused parameters depending the stake type

Reply:
Are we going the unused parameters route?

Reply:
I'm not a huge fan of unused params route